src/components/varsom-avalanche-obs/varsom-avalanche-obs.tsx

```diff
@@ -47,6 +47,26 @@ export class VarsomAvalancheObs {
 
   }
 
+  avalancheStartStopInfo(start: any, stop: any): string {
```

Comment:
Den var ikke dum.
Vanskelig å gjøre dette på en ryddig måte uten at det blir brudd på OnlyOnce, synes du har valgt et bra kompromiss.
Litt pirk:

- jeg mener funksjonsnavn bør inneholde et verb, f.eks.: formatStartStopInfo().
- jeg ser du sender inn start og stop til funksjonen, og du bruker det i if'ene, men du bruker this.HeightStopZone og this.HeightStartZone direkte når du lager tekstene. Foreslår at du velger enten å bruke parametre eller klassevariablene direkte.
- funksjonen kan være privat

Reply:
Takk for tilbakemelding. Trenger jo ikke parametere her, nei.